code/game/machinery/atmoalter/canister.dm

```diff
@@ -193,6 +194,7 @@ update_flag
 
 /obj/machinery/portable_atmospherics/canister/process()
 	src.updateDialog()
+	check_status()
```

Comment:
a better proc name would be nice. also explosions and atmos updates happen in this proc, it should be in process_atmos.